tests/conditional_compilation/conftest.py

```diff
@@ -171,3 +187,64 @@ def save_session_info(pytestconfig, artifacts):
     """Fixture function for saving additional attributes to configuration file."""
     yield
     write_session_info(path=artifacts / SESSION_INFO_FILE, data=pytestconfig.session_info)
+
+
+@pytest.fixture(scope="function")
+def prepare_models(openvino_ref, models, request):
+    """
+    Process models: prepare Open Model Zoo models, skip non-OMZ models.
+    """
+    for model in models:
+        if "OMZ" in model["path"]:
```

Comment:
it's wrong to decide model is from OMZ if "OMZ" is a part of it's path. Model should contain some tag in test config that it's from OMZ to determine that it should be downloaded/converted

Reply:
@vurusovs Fixed, thank you.